lib/storage/data/external/AzureClient.js

```diff
@@ -206,7 +206,7 @@ class AzureClient {
             streamingOptions = azureStreamingOptions;
         } else if (range) {
             // option coming from multipleBackend.upload()
-            const rangeStart = range[0] ? range[0].toString() : undefined;
+            const rangeStart = (range[0] || typeof range[0] === 'number') ? range[0].toString() : undefined;
```

Comment:
could be `range[0] !== undefined`

Reply:
in this case i would need to check !== undefined and !== null, so i prefer to check that we have a number, i could only check the number actually